### PR TITLE
Added NovaSeq X Plus and NextSeq 2000

### PR DESCRIPTION
**Pull request by sofiahag**
There is no run for X Plus added yet, so I'm not sure what it will look like, but we can change the colors later. It should be picked up though, based on https://github.com/SciLifeLab/StatusDB_views/pull/145

Yes, I know some commits were transferred from my other PR, but I removed the code 🤷🏻‍♀️

### CODE REVIEW

File: run_dir/design/flowcell_trend_plot.html [hunk 1]
```diff
@@ -98,13 +98,17 @@ <h1>Flowcells Trend Plot</h1>
...
               </div>
               <div class="form-check form-check-inline">
-                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_nextseq" name="inst_type_filter" value="N">
+                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_nextseq2000" name="inst_type_filter" value="VH">
                 <label class="form-check-label" for="filter_nextseq">NextSeq</label>
               </div>
               <div class="form-check form-check-inline">
-                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_novaseq" name="inst_type_filter" value="A" checked>
+                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_novaseq6000" name="inst_type_filter" value="A" checked>
                 <label class="form-check-label" for="filter_novaseq">NovaSeq</label>
```
Comment (aanil) [inline run_dir/design/flowcell_trend_plot.html:106]: Change label to 6000 as well?

File: run_dir/design/flowcell_trend_plot.html [hunk 2]
```diff
@@ -98,13 +98,17 @@ <h1>Flowcells Trend Plot</h1>
...
               </div>
+              <div class="form-check form-check-inline">
+                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_novaseqxplus" name="inst_type_filter" value="LH" checked>
+                <label class="form-check-label" for="filter_novaseq">NovaSeq X Plus</label>
```
Comment (aanil) [inline run_dir/design/flowcell_trend_plot.html:110]: `filter_novaseq` same as before

File: run_dir/static/js/fc_trend_plot.js [hunk 3]
```diff
@@ -201,10 +203,12 @@ function build_series(data, key, name, display_by, filter_inst_type, filter_inst
...
           }else if (data[d].instrument.indexOf('A') != -1){
-              series_name = "NovaSeq";
-          }else if (data[d].instrument.indexOf('N') != -1){
-              series_name = "NextSeq";
-            }else{
+              series_name = "NovaSeq 6000";
+          }else if (data[d].instrument.indexOf('VH') != -1){
+              series_name = "NextSeq 2000";
+          }else if (data[d].instrument.indexOf('LH') != -1){
+              series_name = "NovaSeq XPlus";
```
Comment (aanil) [inline run_dir/static/js/fc_trend_plot.js:210]: NovaSeq X Plus or NovaSeq XPlus ?

Comment (aanil) [reply run_dir/static/js/fc_trend_plot.js:210]: NovaSeqXPlus is how we've decided it should be named.

Comment (sofiahag) [reply run_dir/static/js/fc_trend_plot.js:210]: Guess we'll go with "NovaSeq X Plus" :-)

File: run_dir/design/flowcell_trend_plot.html [hunk 4]
```diff
@@ -98,12 +98,16 @@ <h1>Flowcells Trend Plot</h1>
...
-                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_nextseq" name="inst_type_filter" value="N">
-                <label class="form-check-label" for="filter_nextseq">NextSeq</label>
+                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_nextseq2000" name="inst_type_filter" value="VH">
+                <label class="form-check-label" for="filter_nextseq2000">NextSeq</label>
               </div>
               <div class="form-check form-check-inline">
-                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_novaseq" name="inst_type_filter" value="A" checked>
-                <label class="form-check-label" for="filter_novaseq">NovaSeq</label>
+                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_novaseq6000" name="inst_type_filter" value="A" checked>
+                <label class="form-check-label" for="filter_novaseq6000">NovaSeq</label>
```
Comment (aanil) [inline run_dir/design/flowcell_trend_plot.html:106]:
```suggestion
                <label class="form-check-label" for="filter_novaseq6000">NovaSeq 6000</label>
```

File: run_dir/design/flowcell_trend_plot.html [hunk 5]
```diff
@@ -98,12 +98,16 @@ <h1>Flowcells Trend Plot</h1>
...
+              </div>
+              <div class="form-check form-check-inline">
+                <input class="form-check-input filter_inst_type" type="checkbox" id="filter_novaseqxplus" name="inst_type_filter" value="LH" checked>
+                <label class="form-check-label" for="filter_novaseqxplus">NovaSeq X Plus</label>
```
Comment (aanil) [inline run_dir/design/flowcell_trend_plot.html:110]:
```suggestion
                <label class="form-check-label" for="filter_novaseqxplus">NovaSeqXPlus</label>
```

File: run_dir/static/js/fc_trend_plot.js [hunk 6]
```diff
@@ -180,11 +182,11 @@ function build_series(data, key, name, display_by, filter_inst_type, filter_inst
...
                 }
-            if (data[d].cver.includes('High')){
-                series_name = "NextSeq High";
+            if (data[d].cver.includes('2000')){
+                series_name = "NextSeq 2000";
                 }
-            if (data[d].cver.includes('Mid')){
-                series_name = "NextSeq Mid";
+            if (data[d].cver.includes('NovaSeqXPlus')){
+                series_name = "NovaSeq X Plus";
```
Comment (aanil) [inline run_dir/static/js/fc_trend_plot.js:189]:
```suggestion
                series_name = "NovaSeqXPlus";
```

File: run_dir/static/js/fc_trend_plot.js [hunk 7]
```diff
@@ -201,10 +203,12 @@ function build_series(data, key, name, display_by, filter_inst_type, filter_inst
...
           }else if (data[d].instrument.indexOf('A') != -1){
-              series_name = "NovaSeq";
-          }else if (data[d].instrument.indexOf('N') != -1){
-              series_name = "NextSeq";
-            }else{
+              series_name = "NovaSeq 6000";
+          }else if (data[d].instrument.indexOf('VH') != -1){
+              series_name = "NextSeq 2000";
+          }else if (data[d].instrument.indexOf('LH') != -1){
+              series_name = "NovaSeq X Plus";
```
Comment (aanil) [inline run_dir/static/js/fc_trend_plot.js:210]:
```suggestion
              series_name = "NovaSeqXPlus";
```

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**